### DISCUSSION TITLE
How Enable built-in translator

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [Eloston/ungoogled-chromium#2115] [Q&A] [answered]
Tell me how to enable the built-in page translator, like in chromium?

Google service is bad! But the built-in Google Translate in chromium is the best in the world.

**Comment by networkException** [comment, marked as answer]
There's no way to do that currently unless you want to modify our patches and build a binary yourself

**Reply by Pantyhose-X** [reply]
> There's no way to do that currently unless you want to modify our patches and build a binary yourself

hey bro
Now there is a really great way to solve this problem！
Contact [FilipePS](https://github.com/FilipePS/Traduzir-paginas-web) to ask him to submit a Pull request.

or add `https://github.com/FilipePS/Traduzir-paginas-web` in chrome://settings/appearance enterr custom web address, Guide the user to install this translation extension

**Reply by networkException** [reply]
I don't think restoring translator functionality using cloud services is a goal of the project, users are free to install the extension themselves.

**Comment by jasursadikov** [comment]
Does "Google Translate" extension does not work properly only on my end? I was curious if it is possible to toggle this functionality.
